### PR TITLE
JA: add site/ja/tutorials/images/transfer_learning_with_hub.ipynb

### PR DESCRIPTION
**Pull request by andromedroid**
I forked from site/en/tutorials/images/transfer_learning_with_hub.ipynb
committed as cc2d9aff156ac01e5dbcec156d6090a03f6895f6

and applied following modification
  - translation to JA.
  - apply tfug/proofreading.
  - reopen on Colab. and saved.

Thanks for all your Help :D

### CODE REVIEW

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 1]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0p7iDOhIcqY2"
+      },
+      "source": [
+        "結果のオブジェクトは `image_batch, label_batch` のペアを返す iterator です。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]
`iterator` -> `イテレーター` or `イテレータ`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 2]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "FUa3YkvhxDLM"
+      },
+      "source": [
+        "画像の帰属については `LICENSE.txt` をご参照ください。\n",
+        "\n",
+        "完璧とは程遠い結果ですが、これらのクラスは（\"daisy\" を除いて）このモデルが学習したクラスではないためというのが合理的な考察です。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]

```
完璧とは程遠い結果ですが、これらのクラスは（\"daisy\" を除いて）このモデルが学習したクラスではないためというのが合理的な考察です。
=====>
完璧とは程遠い結果ですが、このモデルはこれらのクラスのために学習したものではない（ただし、\"daisy\" を除いて）ということを考慮すると、妥当な結果だと言えるでしょう。
```

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 3]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JzV457OXreQP"
+      },
+      "source": [
+        "### headless モデルのダウンロード\n",
+        "\n",
+        "TensorFlow Hub は上位の分類層を含まないモデルも配布しています。これらは転移学習に簡単に利用することができます。\n",
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]
`分類層` -> `識別層`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 4]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0QzVdu4ZhcDE"
+      },
+      "source": [
+        "このモデルは画像毎に 1280 バイト長のベクトルデータを返します:"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [strong suggestion]
`1280 バイト長のベクトルデータ` -> `長さ 1280 のベクトルデータ`

`feature_batch.dtype` is `float32`, I guess it is normally equal to 4 bytes.

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 5]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "58-BLV7dupJA"
+      },
+      "source": [
+        "モデルの学習には `.fit` メソッドを使用します。\n",
+        "\n",
+        "この例を短く保つために学習は 2 epoch だけにします。学習のプロセスを可視化するために、epoch 平均の代わりに各々の batch で個別に loss と accuracy を記録するためのカスタムコールバックを使用します。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]

- `epoch` -> `エポック`
- `batch` -> `バッチ`
- `epoch 平均の代わりに` -> `各エポックでの平均だけではなく`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 6]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Kd0N272B9Q0b"
+      },
+      "source": [
+        "これで、ほんの数回の training の繰り返しでさえ、タスクにおいてモデルが進歩していることが分かりました。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]
`training` -> `訓練`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 7]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "kb__ZN8uFn-D"
+      },
+      "source": [
+        "### 推論結果の確認\n",
+        "\n",
+        "前からプロットをやり直すには、まずクラス名の ordered list を取得します:"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]
`ordered list` -> `リスト`
We can translate `ordered list` as `順序付けされたリスト`, however, I think we can translate it just `リスト`.

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 8]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "4Olg6MsNGJTL"
+      },
+      "source": [
+        "モデルを介して画像 batch を実行し、ID 番号をクラス名に変換します。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]
```
モデルを介して画像 batch を実行し、ID 番号を
=====>
画像のバッチをモデルに入力し、得られた ID を
```

**Comment by AseiSugiyama** [comment]
@andromedroid Could you add the notification of community translation top of the jupyter notebook like [this](https://github.com/tensorflow/docs/blob/master/site/ja/tutorials/quickstart/beginner.ipynb)?

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 9]
```diff
@@ -0,0 +1,999 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "crU-iluJIEzw"
+      },
+      "source": [
+        "[TensorFlow Hub](http://tensorflow.org/hub) は、学習済みモデルコンポーネントを共有する手法のひとつです。学習済みモデルを検索する場合は、こちらの [TensorFlow Module Hub](https://tfhub.dev/) をご覧ください。こちらのチュートリアルでは、これらのデモを行います:\n",
```
[the remark this answers is not in the document]
Comment (kiszk) [reply site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [weak suggestion] `学習済み` -> `訓練済み`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb (hunk 2 above)
Comment (kiszk) [reply site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [weak suggestion] `学習した` -> `訓練された`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb (hunk 3 above)
Comment (kiszk) [reply site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion] `上位` -> `最上位`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 10]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "CtFmF7A5E4tk"
+      },
+      "source": [
+        "新しい分類器のレイヤーのみを変更するために、feature extractor レイヤーの変数をフリーズします。"
```
Comment (kiszk) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]
```新しい分類器のレイヤーのみを変更するために、feature extractor レイヤーの変数をフリーズします。```
->
```feature extractor レイヤーの変数をフリーズすることで、訓練では新しい分類器のレイヤーのみを変更できます。```

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 11]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "OHbXQqIquFxQ"
+      },
+      "source": [
+        "### モデルの学習\n",
```
Comment (kiszk) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [weak suggestion] `学習` -> `訓練`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 12]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "uRcJnAABr22x"
+      },
+      "source": [
+        "## モデルのエクスポート\n",
+        "\n",
+        "学習が完了したので、saved model としてエクスポートします:"
```
Comment (kiszk) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [weak suggestion] `学習` -> `訓練`

**Comment by andromedroid** [comment]
Thanks for your review. I fixed them.
Please confirm them again :)

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 13]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JzV457OXreQP"
+      },
+      "source": [
+        "### headless モデルのダウンロード\n",
+        "\n",
+        "TensorFlow Hub は最上位の識別層を含まないモデルも配布しています。これらは転移学習に簡単に利用することができます。\n",
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]

- `識別` -> `分類`

Sorry, but In this translation, `classification` is translated into `分類` except in this line. I know this is caused by my review, not your mistakes. Could you fix it again?

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 14]
```diff
@@ -0,0 +1,999 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "CtFmF7A5E4tk"
+      },
+      "source": [
+        "feature extractor レイヤーの変数をフリーズして、訓練が新しい分類器のレイヤーのみを変更するようにします。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]

`feature extractor レイヤー` -> `特徴抽出レイヤー` or `特徴抽出層`

File: site/ja/tutorials/images/transfer_learning_with_hub.ipynb [hunk 15]
```diff
@@ -0,0 +1,999 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "58-BLV7dupJA"
+      },
+      "source": [
+        "モデルの訓練には `.fit` メソッドを使用します。\n",
+        "\n",
+        "この例を短く保つために訓練は 2 エポックだけにします。訓練のプロセスを可視化するために、各エポックの平均だけではなく各々のバッチで個別に loss と accuracy を記録するためのカスタムコールバックを使用します。"
```
Comment (AseiSugiyama) [inline site/ja/tutorials/images/transfer_learning_with_hub.ipynb]: [suggestion]

- `loss` -> `損失`
- `accuracy` -> `正確度`

**Comment by andromedroid** [comment]
Sorry about my poor confirmations.
I fixed again them.
So, please confirm them again.

**Comment by AseiSugiyama** [comment]
@andromedroid Great work! LGTM!

**Pull request merged**